<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpotifyApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="lstPlaylist">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>78</y>
      <width>231</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtBuscar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="lstResultadosBusca">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>361</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlay">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>120</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/play.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblResultados">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>57</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultados:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="boxBusca">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>391</width>
      <height>611</height>
     </rect>
    </property>
    <property name="title">
     <string>Buscar MÃºsica</string>
    </property>
    <widget class="QPushButton" name="btnAdicionar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>560</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Adicionar a Playlist &gt;&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="boxPlaylist">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>48</y>
      <width>251</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>Playlist</string>
    </property>
    <widget class="QPushButton" name="btnRemover">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>450</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Remover da Playlist</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLimpaPlaylist">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>510</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Limpar Playlist</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSalvarPlaylist">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>510</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar Playlist</string>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="sldVolume">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>630</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblVolume">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>600</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Volume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProx">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>280</y>
      <width>64</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/next.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnt">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>360</y>
      <width>64</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/prev.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/stop.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblMusicaAtual">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Spotify Player</string>
    </property>
   </widget>
   <zorder>boxPlaylist</zorder>
   <zorder>boxBusca</zorder>
   <zorder>lstPlaylist</zorder>
   <zorder>txtBuscar</zorder>
   <zorder>lstResultadosBusca</zorder>
   <zorder>btnPlay</zorder>
   <zorder>btnBuscar</zorder>
   <zorder>lblResultados</zorder>
   <zorder>sldVolume</zorder>
   <zorder>lblVolume</zorder>
   <zorder>btnProx</zorder>
   <zorder>btnAnt</zorder>
   <zorder>btnStop</zorder>
   <zorder>lblMusicaAtual</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
